Source/WebCore/Modules/indexeddb/IDBTransaction.cpp

```diff
@@ -631,7 +631,7 @@ void IDBTransaction::dispatchEvent(Event& event)
     ASSERT(event.type() == eventNames().completeEvent || event.type() == eventNames().abortEvent);
     m_didDispatchAbortOrCommit = true;
 
-    if (isVersionChange()) {
+    if (isVersionChange() && m_openDBRequest) {
```

Comment:
Do we know why `m_openDBRequest` is nullptr for a version change IDBTransaction? It seems unexpected that m_openDBRequest is nullptr at this point.

If we don't have an explanation for the root cause yet, I would suggest debugging with logs to figure out how `databases()` changes event dispatch and makes m_openDBRequest null. We could simply add a null check but it may cover real issue.